plate_simulation-assets/uijson/plate_simulation.ui.json

```diff
@@ -21,14 +21,14 @@
   "background": {
     "main": true,
     "group": "Basement",
-    "label": "Resistivity (ohm-m)",
+    "label": "Physical property (ohm-m, g/cc, SI)",
```

Comment:
Haven't looked at it yet, but this is probably a bit long for a label.  Maybe we should use SI and then list the specific ones in the tooltip?

Reply:
True

Reply:
Done